src/main/scala/cromwell/Main.scala

```diff
@@ -18,19 +18,18 @@ object Actions extends Enumeration {
 }
 
 object Main extends App {
+  val Properties = System.getProperties
+  val LoggerProperty = "CROMWELL_LOGGER"
+  lazy val Log = LoggerFactory.getLogger("main")
```

Comment:
http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods